src/Microsoft.Identity.Web.Sidecar/Models/DownstreamApiRequest.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Identity.Abstractions;
+
+namespace Microsoft.Identity.Web.Sidecar.Models;
+
+/// <summary>
+/// Represents the inputs to the downstream API endpoint.
+/// </summary>
+public readonly struct DownstreamApiRequest
+{
+    [FromQuery]
+    public string? AgentIdentity { get; init; }
```

Comment:
is this the right name? someone was floating around "blueprint"

Reply:
It is
The blueprint is the app. The agent identity is a principal